features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/AppsAndKeys/ViewToken.jsx

```diff
@@ -169,7 +170,7 @@ class ViewToken extends React.Component {
                     </Typography>
                 </InlineMessage>
                 <div className={classes.epWrapper}>
-                    <Typography className={classes.prodLabel}>
+                    <label for='bootstrap-input' className={classes.prodLabel}>
```

Comment:
`bootstrap-input` we don't use bootstrap 😄

Reply:
Renamed the input ID 😄